lib/predictive_load/active_record_collection_observation.rb

```diff
@@ -1,11 +1,35 @@
 module PredictiveLoad::ActiveRecordCollectionObservation
 
   def self.included(base)
-    ActiveRecord::Relation.send(:include, RelationObservation)
-    ActiveRecord::Base.send(:include, CollectionMember)
-    ActiveRecord::Base.send(:extend, UnscopedTracker)
-    ActiveRecord::Associations::Association.send(:include, AssociationNotification)
-    ActiveRecord::Associations::CollectionAssociation.send(:include, CollectionAssociationNotification)
+    if ActiveRecord::VERSION::MAJOR >= 5
+      ActiveRecord::Relation.include Rails5RelationObservation
+    else
+      ActiveRecord::Relation.include RelationObservation
```

Comment:
renaming both to `ToARelationObservation` / `RecordsRelationObservation` could also be nice ... but I guess this way is less diff 🤷‍♂

Reply:
My goal w/ the naming was to make it as obvious as possible that one was for Rails 4 and one was for Rails 5 so it's easy to update going forward.  I'm not opposed to the names you suggested though since the if statement should be sufficient to drive that point home.